thirteen/two_nieoh.py

```diff
@@ -0,0 +1,37 @@
+import copy
+
+def merge(WAH, WWWAHHH):
+    arr = copy.copy(WAH)
+    brr = copy.copy(WWWAHHH)
+    ret = arr
+    pos = -1
+
+    for i, a in enumerate(arr):
+        if a != 'x':
+            pos += 1
+            last_in_arr = i
+    for i, b in enumerate(brr):
+        pos += 1
+        last_in_brr = i
+    
+    while pos >= 0:
+        while last_in_arr >= 0 and last_in_brr >= 0:
+            if arr[last_in_arr] > brr[last_in_brr]:
+                ret[pos] = arr[last_in_arr]
+                last_in_arr -= 1
```

Comment:
these two line are shared between `if` and `else`. Consider doing them outside of the conditional.